ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -117,17 +123,21 @@ RUN git clone https://github.com/ninja-build/ninja /tmp/ninja \
  && git checkout v1.8.2 \
  && ./configure.py --bootstrap \
  && sudo mv ninja /usr/bin/ninja \
+ && mv misc/bash-completion /home/user/.ninja-bash-completion \
+ && mv misc/zsh-completion /home/user/.ninja-zsh-completion \
+ && echo "\n# Ninja completion helpers." >> /home/user/.bashrc \
+ && echo ". /home/user/.ninja-bash-completion" >> /home/user/.bashrc \
```

Comment:
why not the same in .zshrc?

Reply:
There is no `.zshrc` file yet :/

Reply:
I filed https://github.com/JanitorTechnology/dockerfiles/issues/96 requesting Zsh to be installed and pre-configured in all Docker images.